### PR TITLE
new rushi classes added from rushi branch

### PR DESCRIPTION
**Pull request by rvkathar**
Hi Vaibhav,

can you pls check this PR?

Added rushi java and test page.

Thanks,
Vaibhav

### CODE REVIEW

File: FreeCRMTest/src/main/java/com/crm/qa/pages/RushiPage.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.crm.qa.pages;
+
+import com.crm.qa.base.TestBase;
+
+public class RushiPage extends TestBase {
```
Comment (lavassa05) [inline FreeCRMTest/src/main/java/com/crm/qa/pages/RushiPage.java:5]: Please add all the actions of rushi page.

File: FreeCRMTest/src/main/java/com/crm/qa/pages/RushiPage.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+public static void main(String[] args) {
+	System.out.println("this is rushi page");
```
Comment (lavassa05) [inline FreeCRMTest/src/main/java/com/crm/qa/pages/RushiPage.java:7]: pls add some more end to end test cases

**Review by lavassa05 — CHANGES_REQUESTED** [review]
Hi Rushi,
Please check my review comments and make changes accordingly.

Thanks,
vaibhav

File: FreeCRMTest/src/main/java/com/crm/qa/pages/RushiPage.java (hunk 1 above)
Comment (rvkathar) [reply FreeCRMTest/src/main/java/com/crm/qa/pages/RushiPage.java:5]: Done.Added the logic

File: FreeCRMTest/src/main/java/com/crm/qa/pages/RushiPage.java (hunk 2 above)
Comment (rvkathar) [reply FreeCRMTest/src/main/java/com/crm/qa/pages/RushiPage.java:7]: Done,Added.

**Review by lavassa05 — APPROVED** [review]
PR approved.

**Comment by rvkathar** [comment]
rushi page changes merging with master branch

**Pull request merged**